docs/source/compilation-databases.rst

```diff
@@ -54,11 +54,17 @@ obtain a compilation database for the CPU:
 .. code :: sh
 
     $ mkdir build-cpu
-    $ cmake ../
+    $ cmake -G Ninja ../
     $ ls
 
     CMakeCache.txt  CMakeFiles  Makefile  cmake_install.cmake  compile_commands.json
 
+.. tip::
+    Using the "Ninja" generator is not required, but is often faster and can
```

Comment:
Since it's a tutorial making this recommendation, should we add a short note on how to get ninja (e.g. just `pip install ninja`?)

I don't think we have to advocate for any method specifically persay, just figured that for someone reading this tutorial they probably haven't used Ninja to build anything before. We don't have to add it as a project dependency.

Reply:
Good idea.  Added in https://github.com/intel/code-base-investigator/pull/175/commits/d57a9f6a52a0a25e8262567272474b694cc5604a.